src/Actions/BcgovSettings.php

```diff
@@ -301,4 +344,44 @@ public function bcgov_google_site_name_settings() {
         <p class="description" style="max-width: 120ch;">This value will be used to tell the Google Search index the preferred Site Name. The name default is the Site Title in the WordPress site settings but can be overridden here for finer control of the Google Site Name required to differentiate the site from the inferred Gov.bc.ca site naming in Google organic search results. Note this feature provides an Alternate Site Name of <strong><?php echo esc_html( $domain['host'] ); ?></strong> so it is not necessary to use the domain as the preferred Site&nbsp;Name.</p>
 		<?php
     }
+
+    /**
+     * Sets the Site Documentation URL.
+     *
+     * @since 1.2.20
+     * @return void
+     */
+    public function bcgov_documentation_url_settings() {
+        if ( isset( $_POST['bcgov_documentation_url_nonce'] ) ) {
+
+			if ( wp_verify_nonce( $doc_nonce, 'bcgov_documentation_url_nonce' ) ) {
+				if ( isset( $_POST['bcgov_documentation_url'] ) ) {
+					$documentation_url = sanitize_text_field( $_POST['bcgov_documentation_url'] );
+					update_option( 'bcgov_documentation_url', $documentation_url );
+				}
+			}
+		}
+
+        // Get the current Site Documentation URL setting.
+        $documentation_url = get_option( 'bcgov_documentation_url_settings', '' );
+
+        if ( is_array( $documentation_url ) ) {
```

Comment:
how would this value, ever be an array?

Reply:
get_option can return any type, and array is the only type not automatically transformed into a string. https://developer.wordpress.org/reference/functions/get_option/. Technically it shouldn't be set to anything but a string though, I can remove the check.